### DISCUSSION TITLE
Allow Bulk Insert (not upsert) of platform events via the CLI.

### DISCUSSION THREAD

**Discussion by jjulicher** [forcedotcom/cli#2339] [Ideas]
**Is your feature request related to a problem? Please describe.**

Occasionally we need to bulk insert platform events. However, using the bulk upsert command results in an error:  InvalidJob : No create/update access for object:LeaseChargesBillings__e (even though the file contains only records without any external Id). 

resulting in us to having to manually do a bulk insert via workbench. 

**What are you trying to do**

The ability to bulk insert Platform Events via the CLI. 

**Describe the solution you'd like**

Having either a "data bulk insert" command or having the "data bulk upsert" command recognise platform events by the sobject type extension ('__e') and route the to Bulk API Insert api. 

**Describe alternatives you've considered**

running a curl command to send the file via bulk api. 

**Additional context**

[a comment by an automated account is not shown]

**Comment by mshanemc** [comment]
Normal bulk API lets you specify `Id` as the `--external-id` to do an insert like `sf data upsert bulk --external-id Id`

Your csv doesn't have to have an Id column for this to work.  Can you try it with some __e and see ?

**Comment by jshackell-sfdc** [comment]
Check out version 2.64.6 (October 30, 2024) of Salesforce CLI which contains a new command `data import bulk`.
